control-plane/stor-port/src/types/v0/transport/volume.rs

```diff
@@ -352,6 +352,26 @@ pub struct CreateVolume {
     pub thin: bool,
 }
 
+/// Volume Group related information.
+#[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct VolumeGroup {
+    /// The name of the volume group.
+    pub name: String,
```

Comment:
nit: remove pub and add `fn name(&self) -> &str` ?

Reply:
Right. Added the `fn new()` and `fn name()`